### DISCUSSION TITLE
Execute Custom Script

### DISCUSSION THREAD

**Discussion by rhemasystems** [Mimetis/Dotmim.Sync#1110] [Q&A]
Hi,
Please I have a central server and some few clients that sync data to the Server,its a two way sync.
Is it possible to execute a custom DML script like a raw update statement on the client when they sync data to the server?

Thanks

**Comment by Mimetis** [comment]
I dont understand, can you explain a little bit more ?
you can execute a script before or after sync, obviously

**Reply by rhemasystems** [reply]
During the syncing process,after the data upload by the Client to Server, I want to also  execute an update script  on the client database

**Reply by Mimetis** [reply]
maybe use some interceptors https://dotmimsync.readthedocs.io/Interceptors.html

**Reply by rhemasystems** [reply]
Thanks, will consider that.
But I want to do this update after all the syncing is done, which of the methods can I use to achieve what I want?

**Reply by Mimetis** [reply]
After `SynchronizeAsync()` ?
